### PR TITLE
Updates to the Layer page

### PR DESCRIPTION
**Pull request by dvntucker**
Updates to the layer page as discussed in the sprint:

Add in Title as the second column
Remove workspace column
Removed ? on Enabled
Create Name column with workspace and layer name

### CODE REVIEW

File: src/web/core/src/main/java/org/geoserver/web/data/layer/LayerProvider.java [hunk 1]
```diff
@@ -47,18 +46,20 @@
  */
 @SuppressWarnings("serial")
 public class LayerProvider extends GeoServerDataProvider<LayerInfo> {
-    static final Property<LayerInfo> TYPE = new BeanProperty<LayerInfo>("type",
+    static final Property<LayerInfo> TYPE = new BeanProperty<>("type",
             "type");
 
-    static final Property<LayerInfo> WORKSPACE = new BeanProperty<LayerInfo>(
+    static final Property<LayerInfo> WORKSPACE = new BeanProperty<>(
```
Comment (tbarsballe) [inline src/web/core/src/main/java/org/geoserver/web/data/layer/LayerProvider.java]: After your change, this property is no longer used. Remove it?

**Comment by tbarsballe** [comment]
The header LayerPage.java still need updating (You got halfway there ;-) )

**Comment by dvntucker** [comment]
Oops, little too quick on the pushes.

File: src/web/core/src/main/java/org/geoserver/web/data/layer/LayerPage.java [hunk 2]
```diff
@@ -76,10 +77,12 @@ protected Component getComponentForProperty(String id, IModel<LayerInfo> itemMod
                     return f;
                 } else if(property == SRS) {
                     return new Label(id, SRS.getModel(itemModel));
+                } else if (property == TITLE) {
```
Comment (tbarsballe) [inline src/web/core/src/main/java/org/geoserver/web/data/layer/LayerPage.java]: The `if (property == WORKSPACE)` segment above is probably invalid now if you are no longer displaying workspace?

**Comment by dvntucker** [comment]
Removed the unused BeanProperty and its uses in the LayerPage

**Pull request merged**